python/example/brunel.py

```diff
@@ -33,12 +33,9 @@ class brunel_recipe (arbor.recipe):
     def __init__(self, nexc, ninh, next, in_degree_prop, weight, delay, rel_inh_strength, poiss_lambda, seed = 42):
 
         arbor.recipe.__init__(self)
-        # self.props = arbor.neuron_cable_properties()
-        # self.cat = arbor.default_catalogue()
-        # self.props.register(self.cat)
 
         # Make sure that in_degree_prop in the interval (0, 1]
-        if 0.0>=in_degree_prop>1.0:
+        if not 0.0<=in_degree_prop<1.0:
             print("The proportion of incoming connections should be in the interval (0, 1].")
```

Comment:
This should be `not 0.0<in_degree_prop<=1.0`, for agreement with the error message, and for consistency with the C++ version.

Reply:
:+1: